includes/class-wcs-att-integrations.php

```diff
@@ -226,9 +501,139 @@ private static function has_individually_priced_bundled_contents( $product ) {
 		}
 	}
 
+	/**
+	 * Set the active bundle scheme on a bundled item.
+	 *
+	 * @param  WC_Bundled_Item    $bundled_item
+	 * @param  WC_Product_Bundle  $bundle
+	 */
+	public static function set_bundled_item_scheme( $bundled_item, $bundle ) {
+
+		// Callable since PB 5.2.4.
+		if ( is_callable( array( $bundled_item, 'get_product' ) ) ) {
+
+			$having = array(
+				'price',
+				'regular_price'
+			);
+
+			$what = array(
+				'min',
+				'max'
+			);
+
+			if ( $bundled_product = $bundled_item->get_product() ) {
+				self::set_bundled_product_subscription_schemes( $bundled_product, $bundle );
+			}
+
+			foreach ( $having as $price ) {
+				foreach ( $what as $min_or_max ) {
+					if ( $bundled_product = $bundled_item->get_product( array( 'having' => $price, 'what' => $min_or_max ) ) ) {
+						self::set_bundled_product_subscription_schemes( $bundled_product, $bundle );
+					}
+				}
+			}
+		}
+	}
+
+	/**
+	 * Calculates bundle container item subtotals.
+	 *
+	 * TODO: Maybe calculate this from recurring carts to account for discounts?
```

Comment:
Yes, feature/coding debt needs to be logged in the issue tracker. I'm not even sure when I added this. I usually add TODOs while working and then `grep` for them on the way to getting everything done, then make sure nothing is left.

It's been such a long time since I wrote this that I no longer remember what it is about. The shame!